proposals/nnnn-atf_build_using_cmake.md

```diff
@@ -0,0 +1,43 @@
+# ATF build using CMake
+
+* Proposal: [SDL-NNNN](nnnn-atf_build_using_cmake.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+This proposal offers using cmake build system for compilation ATF project instead of qmake. 
+
+## Motivation
+
+Extending sdl_atf with new dependencies and new code requires seamless integration with build system.  
+Existing qmake and make systems are not expandable and adding any supplementary components is rather complicated.  
+CMake will be able to provide clear dependency structure, seamless integration with 3rd party libraries, easy expandable project structure.
+
+## Proposed solution
+
+This proposed change is to create CMakeLists.txt with all required dependency description.
```

Comment:
Please add **s** to description

This proposed change is to create CMakeLists.txt with all required dependency descriptions.